xbmc/cores/VideoPlayer/DVDCodecs/Overlay/DVDOverlayCodecCCText.cpp

```diff
@@ -123,6 +123,14 @@ OverlayMessage CDVDOverlayCodecCCText::Decode(DemuxPacket* pPacket)
   return m_pOverlay ? OverlayMessage::OC_DONE : OverlayMessage::OC_OVERLAY;
 }
 
+void CDVDOverlayCodecCCText::PostProcess(std::string& text)
+{
+  // The data that come from InputStream could contains \r chars
+  // we have to remove them all because it causes to display empty box "tofu"
+  //! @todo This must be removed after the rework of the CC decoders
```

Comment:
Is this todo still valid?

Reply:
CC decoders will one day have to be updated then this is no longer needed,
from what i have seen `\r` can be managed directly from CC decoders then avoiding add this char